### PR TITLE
35 Day to DayIndex Conversion 

### PR DESCRIPTION
**Pull request by Fard-Faru**
## Modification: Changed Position of Day to DayIndex Conversion

Description:  
Modified the switch statements to Controllers rather than InputData

### Important Notes:
Weak Warning for Duplicate Data, group consensus, keep the duplicate data to attempt to adhere to single responsibility principles (we should mention this in the docs)

### CODE REVIEW

File: src/main/java/create_event_use_case/CreateEventController.java [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package create_event_use_case;
 
+import create_task_use_case.CreateTaskController;
```
Comment (missionStrawberry) [inline src/main/java/create_event_use_case/CreateEventController.java:3]: remove import statement.

**Review by missionStrawberry — CHANGES_REQUESTED** [review]
Looks good besides one import statement. Thanks for fixing this for both these use cases Fardin!

**Comment by missionStrawberry** [comment]
Accidentally checked out to branch 35 instead of 61 for making changes to the view model, and group consensus was that because it was a minor issue it would be more hassle to undo the push than just allow the two issues to be linked.

File: src/main/java/create_event_use_case/CreateEventController.java (hunk 1 above)
Comment (Fard-Faru) [reply src/main/java/create_event_use_case/CreateEventController.java]: resolved

**Review by missionStrawberry — APPROVED** [review]
Looks good!

**Review by POPPININ — APPROVED** [review]
Looks good!

**Review by scissorkhizer — APPROVED** [review]
Looks good! Thanks for making the change from a string input of Day to an int input of dayIndex for additional compatibility with the ViewModel!

**Pull request merged**